src/main/java/github/scarsz/discordsrv/util/DiscordUtil.java

```diff
@@ -508,8 +508,8 @@ public static String convertRoleToMinecraftColor(Role role) {
      * @param member The Member to retrieve the roles of
```

Comment:
Update javadoc comments accordingly

Reply:
Changed the comment to reflect the parameter is now a `List` of `Role`s